tests/test_iotedgemodule.py

```diff
@@ -0,0 +1,27 @@
+import os
+import pytest
+
+root_dir = os.getcwd()
+module_name = "iotedgeFilterModule"
+file_list = ["requirements.txt", "module.json", "main.py", "Dockerfile.arm32v7",
+             "Dockerfile.amd64.debug", "Dockerfile.amd64", ".gitignore"]
+
+
+@pytest.fixture(scope="module")
+def test_install_iotedgemodule(request):
+    installmodulecommand = "cookiecutter --no-input https://github.com/Azure/cookiecutter-azure-iot-edge-module module_name=" + \
+        module_name + " image_repository=localhost:5000/"+module_name+" --checkout master"
```

Comment:
test against master?

Reply:
Using local bits from branch which is created by dev for testing. Updated code.